solutions/page_object/pages/base_page.py

```diff
@@ -23,6 +23,26 @@ def is_element_present(self, locator):
             return False
         return True
 
+    def is_not_element_present(self, locator, timeout=4):
+        try:
+            WebDriverWait(self.browser, timeout).until(
+                ec.presence_of_element_located(locator)
+            )
```

Comment:
Не будет ли так проще читаться?

```suggestion
            )
            return False
```